src/ngraph/op/subtract.hpp

```diff
@@ -50,9 +50,43 @@ namespace ngraph
                 virtual void generate_adjoints(autodiff::Adjoints& adjoints,
                                                const NodeVector& deltas) override;
             };
-        }
+        } // namespace v0
+
+        namespace v1
+        {
+            /// \brief Elementwise subtraction operation.
+            class Subtract : public util::BinaryElementwiseArithmetic
+            {
+            public:
+                NGRAPH_API
+                static constexpr NodeTypeInfo type_info{"Subtract", 1};
+                const NodeTypeInfo& get_type_info() const override { return type_info; }
+                Subtract()
+                    : util::BinaryElementwiseArithmetic(AutoBroadcastSpec::NONE)
```

Comment:
Are you sure `AutoBroadcastType::NONE` this is correct value since default value for Broadcasting in the second constructor we are using `AutoBroadcastType::NUMPY`?
```suggestion
                    : util::BinaryElementwiseArithmetic(AutoBroadcastSpec::NUMPY)
```

Reply:
Changed to `NUMPY`.